lib/runner.js

```diff
@@ -65,6 +65,10 @@ optionChain(chainableMethods, function (opts, title, fn) {
 		throw new TypeError('Expected a function. Use `test.todo()` for tests without a function.');
```

Comment:
Yea that should be fine.

Would be good to have a test for this new behavior as well.